content/example/ex020/draw-model.sh

```diff
@@ -0,0 +1,18 @@
+gmt set MAP_TICK_LENGTH_PRIMARY -0.1c
+model=model.txt
+PS=model.ps
+gmt psbasemap -R0/10/0/50 -JX8c/-10c -Bx1+l"velocity (km/s)" -By10+l"depth (km)" -BWN -K -P > $PS
```

Comment:
`velocity`-> `Velocity`. 
`depth` -> `Depth`

Reply:
Most journals require that the first letter of the first word of labels are uppercase. 

For example, `Travel time residual`, only `T` is uppercase.